.ci/ci.yml.tmpl

```diff
@@ -1,12 +1,27 @@
-{% set chef_modules = ['auth', 'compute', 'sql', 'storage', 'spanner', 'container', 'dns'] %}
-{% set puppet_modules = ['auth', 'bigquery', 'compute', 'sql', 'storage', 'spanner', 'container', 'dns', 'pubsub', 'resourcemanager'] %}
+{% set chef_modules = ['_bundle', 'auth', 'compute', 'sql', 'storage', 'spanner', 'container', 'dns'] %}
+{% set puppet_modules = ['_bundle', 'auth', 'bigquery', 'compute', 'sql', 'storage', 'spanner', 'container', 'dns', 'pubsub', 'resourcemanager'] %}
+{% set puppet_custom_submodules = {
+    '_bundle': 'puppet-google'
+  }
+%}
+{% set chef_custom_submodules = {
+    '_bundle': 'chef-google'
+  }
+%}
 {% set terraform_enabled = true %}
 {% set ansible_enabled = true %}
 {% macro names_as_list(repo, names) -%}
 {% for name in names %}
 build/{{repo}}/{{name}} 
 {%- endfor %}
 {% endmacro -%}
+{% macro submodule_name(repo, submodule_hashes, prefix) -%}
```

Comment:
This looks confusing - if it can't be made pretty, can we comment it so it makes sense?

Reply:
Made this much simpler now that I'm not trying to generalize it.